controllers/rcj_soccer_referee_supervisor/referee/referee.py

```diff
@@ -125,44 +121,46 @@ def kickoff(self, team: Optional[str] = None):
 
         robot_name = self.reset_team_for_kickoff(team)
 
-        self.log.event(
+        self.eventer.event(
             supervisor=self,
             type=GameEvents.KICKOFF.value,
-            robot_name=robot_name,
-            msg=f"Robot {robot_name} is kicking off."
+            payload={
+                "robot_name": robot_name,
+                "team": team,
```

Comment:
@Adman this is my bad, but to be consistent, let's call this `team_name` as well (just as above)

Reply:
Renamed.